aws-android-sdk-cognitoidentityprovider-test/build.gradle

```diff
@@ -2,14 +2,14 @@ apply plugin: 'com.android.library'
 apply plugin: 'devicefarm'
 
 android {
-    compileSdkVersion 27
+    compileSdkVersion 28
 
     defaultConfig {
         minSdkVersion 15
-        targetSdkVersion 27
+        targetSdkVersion 28
         versionCode 1
         versionName '1.0'
-        testInstrumentationRunner 'android.support.test.runner.AndroidJUnitRunner'
+        testInstrumentationRunner 'androidx.test.runner.AndroidJUnitRunner'
```

Comment:
According to your PR description, `AndroidJUnit4Runner` is no longer used, so can this be removed?  (if so, remove it from all the build.gradle files in this PR)

Reply:
They are similarly named. This is the AndroidJUnit___Runner___; the old one was the AndroidJUnit__4Runner__. It can be selected by stating a [`testInstrumentationRunner`](https://google.github.io/android-gradle-dsl/current/com.android.build.gradle.internal.dsl.DefaultConfig.html#com.android.build.gradle.internal.dsl.DefaultConfig:testInstrumentationRunner) in Gradle, whereas the old one with the "4" in its name had to be elected in an `@RunWith` annotation.